### ISSUE TITLE
Crash when updating attribute with empty JSON Object

### ISSUE THREAD

**Issue by fgalan** [telefonicaid/fiware-orion#3995]
Doing an update in existing entity like this:

```
curl 'localhost:1026/v2/entities/E/attrs?type=T' -H 'content-type: application/json' --data-raw '{
        "A": {
            "type": "object",
            "value": {}
	}
}'
```

Makes de broker to crash.

I think this bug was introduced in Orion 3.3.0 version. Previous versions are not affected.

**Comment by fgalan** [comment]
Probably also happens with `"value": []`

**Comment by fgalan** [comment]
Fixed in PR #3996 (master) and #3997 (release/3.3.0)
